ops.h

```diff
@@ -353,7 +353,7 @@ static HWY_NOINLINE HWY_MAYBE_UNUSED void RMSNorm(
     float* HWY_RESTRICT out, size_t size) {
   constexpr float eps = 1e-6f;
   float ss = SquaredL2(x, size);
-  ss = 1.0f / sqrtf(ss / static_cast<int>(size) + eps);
+  ss = 1.0f / sqrtf(ss / static_cast<float>(size) + eps);
```

Comment:
FYI this is actually a slight pessimization, it is cheaper to convert int32 to float than u64 -> float. If we want to be explicit, `cast<float>(cast<int>(size))` would be ideal.

Reply:
What do you think of adding a function like FloatFromSmallInt? Could be a template supporting types such as size_t, and would avoid replicating these two casts in various places.

Thanks BTW for adding const and fixing these issues :)

Reply:
I was thinking of creating a safe_cast function to ensure casting without causing overflow.

Reply:
Sounds good. How about we only do range checks (if any) in debug builds, to avoid overhead? You can use HWY_DASSERT for such things.